client/src/common/components/ModalForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import Overlay, { OverlayStyleType } from 'common/components/Overlay';
+import CreationForm, { CreationFormType } from 'common/components/CreationForm';
+
+const ModalForm = ({
+  defaultDescription,
+  defaultName,
+  label,
+  onCancel,
+  onSubmit
+}) => (
```

Comment:
shouldn't ModalForm receive an  additional property to descide whether to display creation form or not?